File: 10101.py
x = int(input())
y = int(input())
z = int(input())

if x==60 and y==60 and z==60:
    print("Equilateral")
elif x+y+z==180 and ((x==y or x==z or y==z)):
    print("Isosceles")
elif x+y+z==180 and ((x!=y or x!=z or y!=z)):
    print("Scalene")
elif x+y+z!=180:
    print("Error")

File: 10101.c
#include <stdio.h>

int main() {

  int x,y,z;
  scanf("%d", &x);
  scanf("%d", &y);
  scanf("%d", &z);

  if((x==60) && (y==60) && (z==60)) {
    printf("Equilateral");
  } else if((x+y+z==180) & ((x==y)||(x==z)||(y==z))) {
    printf("Isosceles");
  } else if((x+y+z==180) & ((x!=y)||(x!=z)||(y!=z))) {
    printf("Scalene");
  } else if(x+y+z!=180) {
    printf("Error");
  }
}
